test/linter/test_buf_lint.vader

```diff
@@ -0,0 +1,22 @@
+Before:
+  call ale#assert#SetUpLinterTest('proto', 'buf-lint')
```

Comment:
The SetUpLInterTest() function uses filename matching to setup the linter. Since the filename is "proto/buf_lint.vim" the second argument here should be "buf_lint" or ALE won't find the linter.